### PR TITLE
add scoring mechanisms page

### PR DESCRIPTION
**Pull request by jmcook1186**
Adds page on scoring mechanisms to 'Major Concepts' section

### CODE REVIEW

File: pages/building-with-passport/major-concepts/scoring-mechanisms.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### Unique Humanity
+
+Unique Humanity assigns a weight to each individual Stamp. The score is simply the sum of the Stamp weights. The Stamp weights are set by Gitcoin
+and can be viewed on the [Passport Github](https://github.com/gitcoinco/passport-scorer/blob/main/api/scorer/settings/gitcoin_passport_weights.py).
+
+The intended use for this is that an app can set a threshold that can be used to gate access to certain content or functions. For example, you might determine 
+that your users have to have a Passport score of 20 to access your platform. A threshold of 20 is generally considered to be a sensible default for removing
+bots and fake users from your application. You can choose to be more or less lenient depending on your specific use case.
+
+With the Unique Humanity Sorer, the threshold is not applied for you on the server. You receive a numeric value to apply as you see fit.
```
Comment (lebraat) [inline pages/building-with-passport/major-concepts/scoring-mechanisms.mdx:23]: Scorer is misspelled

File: pages/building-with-passport/major-concepts/scoring-mechanisms.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Custom scoring mechanisms
+
+You are also welcome to apply your own scoring mechanism to raw Stamp data. The built-in Scorers are executed on the Gitcoin server. You can also 
+access a user's Stamps directly and implement your own algorithm to determine the Passport value. You can do this with Stamp data retrieved from 
+the Gitcoin server or onchain Stamps. An example of client-side custom scoring is provided in our [onchain Stamps](/building-with-passport/tutorials/integrating-onchain-stamp-data.md) tutorial.
+
+
+
+## Stamp deduplication
```
Comment (lebraat) [inline pages/building-with-passport/major-concepts/scoring-mechanisms.mdx:52]: Can we please link to the Stamp deduplication page from this section?

**Review by lebraat — CHANGES_REQUESTED** [review]
Looks great overall! 

A follow-up task will be to audit a few docs pages for places that we link to 'scorer types' (currently links to Getting access) and relink to this page.

File: pages/building-with-passport/major-concepts/scoring-mechanisms.mdx (hunk 1 above)
Comment (jmcook1186) [reply pages/building-with-passport/major-concepts/scoring-mechanisms.mdx:23]:
```suggestion
With the Unique Humanity Scorer, the threshold is not applied for you on the server. You receive a numeric value to apply as you see fit.
```

**Pull request merged**